src/main/java/org/alfresco/heartbeat/ModelUsageDataCollector.java

```diff
@@ -37,6 +37,15 @@
 
 import java.util.*;
 
+/**
+ * This class collects model usage data for HeartBeat.
+ * <br>
+ * <b>Collector ID:</b> acs.repository.usage.model
+ * <br>
+ * <b>Data points:</b> numOfActiveModels, numOfActiveTypes, numOfActiveAspects
```

Comment:
Each of the fields needs a description:
`
<ul>
  <li><b>numOfActiveModels:</b> blah blah </li>
  <li><b>numOfActiveTypes:</b> blah blah </li>
`
Each of the data points provides some value to someone, presumably.